crates/matrix-sdk-base/src/sliding_sync/http.rs

```diff
@@ -266,6 +266,12 @@ pub mod simplified_msc3575 {
 
     use super::{msc3575 as ss, From};
 
+    impl From<Response> for Response {
+        fn from(value: Response) -> Self {
+            value
+        }
+    }
+
```

Comment:
Yeah, it was required to make `http::Into::<http::simplified_msc3575>::into` to work on both `Response` types (from `simplified_msc3575` and `msc3575`). That's to mimic [the reflexive implementation of `std::convert::From`](https://doc.rust-lang.org/std/convert/trait.From.html#generic-implementations). Note that this has been removed since we do the conversion between these types directly in `ruma` with `std::convert::From` and `std::convert::Into`.